IdentityCore/src/MSIDConstants.h

```diff
@@ -86,7 +86,27 @@ extern NSString * _Nonnull const MSID_BROKER_APP_BUNDLE_ID;
 extern NSString * _Nonnull const MSID_BROKER_APP_BUNDLE_ID_DF;
 extern NSString * _Nonnull const MSID_BROKER_MAX_PROTOCOL_VERSION;
 extern NSString * _Nonnull const MSID_BROKER_PROTOCOL_VERSION_KEY;
+/**
+ This protocol is being used in ADAL for URL scheme requests to broker.
+ */
+extern NSInteger const MSID_BROKER_PROTOCOL_VERSION_2;
+/**
+This protocol is being used in MSAL for URL scheme requests to broker.
+*/
+extern NSInteger const MSID_BROKER_PROTOCOL_VERSION_3;
+/**
+This protocol is being used in MSAL for XPC requests to SSO extension.
```

Comment:
will this be also used for MSAL in combination when SSO extension exists but we call broker through URL schemes? My assumption is that we would not return RT in that case, so I'm not clear what protocol version we should use there.

Reply:
We agreed to use current URL scheme protocol with extra flag. Will address this in separate pr.